source/install/config-ssl-http2-apache2.rst

```diff
@@ -51,6 +51,7 @@ When opened, edit it to look something like the following:
 		</Location>
 
 		# Generated by Certbot
+		SSLEngine On
```

Comment:
On my system, SSL Engine On is enabled in `/etc/letsencrypt/options-ssl-apache.conf`, which is included three lines below.

Reply:
However, since this is the ssl configuration documentation, I does make sense to explicitly add `SSLEngine On`